src/databases/ffp/avtffpFileFormat.C

```diff
@@ -1761,7 +1792,7 @@ avtffpFileFormat::ReadFile(void)
                     unvhandle = fopen(unvfile.c_str(), "r");
                     bool flipflop = false ;
                     int len = 2048; // Longest line length
-                    char buf[len]; // A line length
+                    char buf[2048]; // A line length
```

Comment:
If we change the declaration above for `int len = 2048` to `int const len = 2048`, does that correct Windows compilation for line 1795 where `len` is used as size of an automatic array? I think I may see a similar pattern around line 1245 that appears to have passed compilation ok on Windows.

Reply:
Just curious, what's the difference between 'const int len = 2048' and 'int const len = 2048'. I've never seen it the way you've written it.

Reply:
Yes, making 'len' constant will fix it. Elsewhere in the code it is 'const int len' though, not 'int const len'.

Reply:
> Just curious, what's the difference between 'const int len = 2048' and 'int const len = 2048'. I've never seen it the way you've written it.

There is no difference in the type. I do it for *readability*.

Somewhere in the past I learned a practice where you can write the qualifiers of a type such that when you read them in reverse order, it makes better English sense of what the type is. For example, `char const * const *p` means that `p` is a pointer (first `*`) to a const-pointer (first `const` and second `*`) to a constant char (second `const`). Or that `int const len` means that `len` is a constant `int`. So, I've always done it that way since then.